pkg/webservice/handlers_test.go

```diff
@@ -1500,7 +1500,7 @@ func TestUsersAndGroupsResourceUsage(t *testing.T) {
 	assert.NilError(t, err, "Get Groups Resource Usage Handler request failed")
 
 	var groupsResourceUsageDao []*dao.GroupResourceUsageDAOInfo
-	var expGroupsResourceUsageDao []*dao.GroupResourceUsageDAOInfo
+	expGroupsResourceUsageDao := []*dao.GroupResourceUsageDAOInfo{}
```

Comment:
pardon me, is this change related to PR? or it is a potential issue?

Reply:
Yes it's related. Since the response from the code you mentioned is now empty array instead of nil as we changed the behavior in this pr.

IMO, the test should be adjusted as follows: set up a scenario where the group resource usage is not empty, ensuring that there is content in groupsResourceUsageDao. Otherwise, it appears to be duplicating the functionality of TestEmptyGroupsResourceUsageHandler.

Reply:
> IMO, the test should be adjusted as follows: set up a scenario where the group resource usage is not empty, ensuring that there is content in groupsResourceUsageDao. Otherwise, it appears to be duplicating the functionality of TestEmptyGroupsResourceUsageHandler.

that makes sense to me. Please make sure the test is NOT duplicate to `TestEmptyGroupsResourceUsageHandler`

Reply:
@brandboat Empty response is one of the basic test cases while hitting those API's. So, can we add the test related changes in `TestUsersAndGroupsResourceUsage` itself instead of two new & separate test methods?

Reply:
Already addressed comments in the latest commit, many thanks !